File: java/src/test/java/com/alxarafe/domain/topology/valueobject/LocationStatusTest.java
package com.alxarafe.domain.topology.valueobject;

import com.alxarafe.app.domain.topology.valueobject.LocationStatus;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class LocationStatusTest {

    @Test
    void values() {
        assertEquals(LocationStatus.ACTIVE, LocationStatus.valueOf("ACTIVE"));
        assertEquals(LocationStatus.BLOCKED, LocationStatus.valueOf("BLOCKED"));
        assertEquals(LocationStatus.DISABLED, LocationStatus.valueOf("DISABLED"));
        assertEquals(3, LocationStatus.values().length);
    }
}

File: php/tests/Unit/Domain/Topology/ValueObject/LocationStatusTest.php
<?php

declare(strict_types=1);

namespace Tests\Unit\Domain\Topology\ValueObject;

use Alxarafe\App\Domain\Topology\ValueObject\LocationStatus;
use PHPUnit\Framework\TestCase;

final class LocationStatusTest extends TestCase
{
    public function testValues(): void
    {
        self::assertSame('ACTIVE', LocationStatus::ACTIVE->value);
        self::assertSame('BLOCKED', LocationStatus::BLOCKED->value);
        self::assertSame('DISABLED', LocationStatus::DISABLED->value);
    }
}
